ecl/hql/hqlparse.cpp

```diff
@@ -548,7 +549,10 @@ void HqlLex::checkSignature(const attribute & dummyToken)
         return;
     try
     {
-        yyParser->gpgSignature.setown(::checkSignature(text->length(), text->getText()));
+        StringBuffer signer, errmsg;
+        if (!queryCodeSigner().verifySignature(text->getText(), signer, errmsg))
+            throw makeStringExceptionV(0, "gpg: signature not verified");
```

Comment:
should this use errmsg?

Reply:
I've changed verifySignature to remove errmsg argument.  It will now throw exceptions instead of return error messages in errmsg.